CANtactCore/src/org/cantact/core/CantactDevice.java

```diff
@@ -18,7 +18,7 @@
     private int speedMode = 0;
 
     public CantactDevice(String deviceName) {
-        if (deviceName != "")
+        if (deviceName.equals(""))
```

Comment:
Wouldn't this mean it only opens the device if it is blank? Shouldn't this have a ! in front?